buffer/line.go

```diff
@@ -6,12 +6,14 @@ import (
 
 type Line struct {
 	wrapped bool // whether line was wrapped onto from the previous one
+	nobreak bool // true if no line break at the beginning of the line
```

Comment:
It would be clearer to read the changes in this PR if the sense of this was changed so this was just `break`.
`setNoBreak(true)` is harder to think about than `setBreak(false)`

Reply:
well, setBreak sounds weird to me. Need to specify what kind of break, maybe )